### PR TITLE
Property to avoid of having repeated dates per month  except in the first one. 

### PR DESCRIPTION
**Pull request by hsuarezFolder**
Property to avoid of having repeated dates per month except in the first one. 
image_2019_11_26T16_10_00_927Z

image_2019_11_26T16_09_24_182Z

### CODE REVIEW

File: src/components/Calendar.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ type StateType = {
   initialScrollIndex: number,
   startDate: ?Date,
   endDate: ?Date,
+  disableOffsetDays: boolean,
```
Comment (maggialejandro) [inline src/components/Calendar.js]: It shouldn't be here

File: src/components/Month/index.js [hunk 2]
```diff
@@ -101,24 +101,27 @@ type Props = {
   disableRange: boolean,
   disabledDays: { [key: string]: any },
   extraData: any,
+  disableOffsetDays: boolean,
```
Comment (maggialejandro) [inline src/components/Month/index.js]: Should be optional

File: src/components/Month/index.js [hunk 3]
```diff
@@ -101,24 +101,27 @@ type Props = {
...
 };
 
 const getDayList = (props: Props) => {
   const {
-    month: { monthNumber, year },
+    month: { monthNumber, year, indexList },
```
Comment (maggialejandro) [inline src/components/Month/index.js]: indexList is never used

File: src/components/Month/utils.js [hunk 4]
```diff
@@ -65,55 +68,70 @@ export function getDaysOfMonth(
...
+        isMonthDate,
+        isActive,
+        isStartDate,
+        isEndDate,
+        isOutOfRange,
+        isVisible:
+          isOnSelectedRange &&
+          isMonthDate &&
+          !disabledDays[moment(date).format('YYYY-MM-DD')],
+        shouldRender: true,
```
Comment (maggialejandro) [inline src/components/Month/utils.js]: Maybe we can name it isHidden (and invert its value), could make more sense

File: src/types/index.js [hunk 5]
```diff
@@ -78,6 +78,7 @@ export type CalendarType = {
   viewableItemsChanged?: (ViewableItemsType) => void,
   calendarListRef?: (ref: React.ElementRef<typeof FlatList>) => void,
   disabledDays: { [key: string]: any },
+  disableOffsetDays: boolean,
```
Comment (maggialejandro) [inline src/types/index.js]: Should be optional

**Comment by maggialejandro** [comment]
Hi @hsuarezFolder 
Great PR :tada:

I made some comments.
Also, I'm close to finish migrate the codebase to typescript. Let me know if you are willing to made a new PR (the branch it's called **ts**). I'll delay the merge for now

Thank you 👍

**Comment by hsuarezFolder** [comment]
Hi @maggialejandro,

thanks for taking a look, I made the required changes and I will make a new PR in the ts branch.

regards

**Comment by maggialejandro** [comment]
@hsuarezFolder thank you!

**Pull request merged**